src/proof/arith_proof.h

```diff
@@ -97,6 +106,15 @@ class LFSCArithProof : public ArithProof {
    */
   static void printRational(std::ostream& o, const Rational& r);
 
+  /**
+   * Print an integer in LFSC format.
+   *        e.g. 5 or (~ 1)
+   *
+   * @param o ostream to print to.
+   * @param i the integer to print
+   */
+  static void printInteger(std::ostream& o, const Integer& i);
```

Comment:
I am not sure these two functions need to be static, maybe they can be functions that are external to the class (I am following [these](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#nonmember-static-member-and-global-functions) instructions here). On the other hand, the rest of the similar printing functions are static so it would be weird and not uniform to change only this one. I'll leave it for you to decide, just wanted to mention this in case you haven't thought of that.

Reply:
That's a good question. I hadn't noticed that. I would prioritize uniformity over style guide compliance.

Perhaps I'll submit a PR fixing the style later.